src/applications/claims-status/components/claim-status-tab/OpenClaimDecisionLetterAlert.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Link } from 'react-router-dom-v5-compat';
+
+export default function OpenClaimDecisionLetterAlert() {
+  return (
+    <va-alert class="vads-u-margin-bottom--4" status="info">
+      <h2 id="claims-alert-header" slot="headline">
+        You have a decision letter for part of this claim
+      </h2>
+      <p>
+        <strong>Another part of this claim is still in progress</strong>
+      </p>
+      <p>
+        You can download your decision letter online now. You can also get other
+        letters related to your claims.
+      </p>
+      <p className="vads-u-margin-y--0">
+        We’ll also send you a copy of your decision letter by mail. It within 10
```

Comment:
The wording on the second sentence doesn't make sense, I think there might be some missing words there